roles/federation-demo/tasks/postgres.yml

```
@@ -0,0 +1,47 @@
+---
+- name: Make sure psycopg2 is installed
+  command: pip install psycopg2==2.7.5 --ignore-installed
+
+- name: Send file
+  copy:
+    src: private_files/setup_db.sql
+    dest: "{{federation_db.db_script_path}}"
```

```suggestion
    dest: "{{federation_db.db_script_path}}"
    backup: yes
```

https://docs.ansible.com/ansible/2.9/modules/copy_module.html#parameter-backup